tests/ut_mdesktopentry.cpp

```diff
@@ -396,6 +395,7 @@ void UtMDesktopEntry::localization()
 
     if (!lang.isEmpty()) {
         qputenv("LANG", lang.toLocal8Bit());
+        qputenv("LC_ALL", lang.toLocal8Bit());
```

Comment:
Yea, LC_ALL should have precedence over any other (which should make LANG above redundant too), but code should still work with LANG only, being used if no proper LC_* is found. Jolla uses only LANG for setting device locale.